api/server/router/network/network_routes.go

```diff
@@ -85,6 +86,15 @@ func (n *networkRouter) getNetwork(ctx context.Context, w http.ResponseWriter, r
 	}
 
 	term := vars["id"]
+	var (
+		verbose bool
+		err     error
+	)
+	if v := r.URL.Query().Get("verbose"); v != "" {
+		if verbose, err = strconv.ParseBool(v); err != nil {
+			verbose = true
+		}
```

Comment:
I think this should return an error if `err` is non-nil. See, for example, https://github.com/docker/docker/blob/master/api/server/router/swarm/cluster_routes.go#L78